./error_fun_ret2.nit:17,5--8: Control error: Reached end of function (a 'return' with a value was expected).
